spinta/cli/inspect.py

```diff
@@ -37,33 +57,767 @@ def inspect(
     auth: Optional[str] = Option(None, '-a', '--auth', help=(
         "Authorize as a client"
     )),
+    priority: str = Option('manifest', '-p', '--priority', help=(
+        "Merge property priority ('manifest' or 'external')"
+    ))
 ):
     """Update manifest schema from an external data source"""
+    if priority not in ['manifest', 'external']:
+        raise Exception(f"\'{priority}\' priority doesnt not exist, there can only be \'manifest\' or \'external\'")
```

Comment:
I think, this should not be an exception, but a simple error message, printed with `print`.